File: Python/Sort.py
def sort(self):
    n = self.rear - self.front + 1
    if n <= 1:
        return  # Queue is already sorted

    swapped = True
    while swapped:
        swapped = False
        for i in range(self.front, self.rear):
            if self.array[i] > self.array[i + 1]:
                # Swap elements
                self.array[i], self.array[i + 1] = self.array[i + 1], self.array[i]
                swapped = True

File: C++/Sort.cpp
void sort() {
    int n = rear - front + 1;
    if (n <= 1) {
        return;  // Queue is already sorted
    }

    int swapped;
    int temp;
    do {
        swapped = 0;
        for (int i = front; i < rear; i++) {
            if (array[i] > array[i + 1]) {
                // Swap elements
                temp = array[i];
                array[i] = array[i + 1];
                array[i + 1] = temp;
                swapped = 1;
            }
        }
    } while (swapped);
}
